src/python/WMCore/Services/pycurl_manager.py

```diff
@@ -55,10 +55,67 @@
 import pycurl
 from io import BytesIO
 import http.client
-from urllib.parse import urlencode
+from urllib.parse import urlencode, urlparse, ParseResult
 
 from Utils.Utilities import encodeUnicodeToBytes
 
+
+def portForward(port):
```

Comment:
This decorator would be better to go under the Utils package (with a proper documentation ;))

Reply:
Done